Site header

kateryna-klimchuk opened the pull request:
Сделала хедер, но стили кнопок и свг-иконки мобильного меню (они у меня есть, но жду свг-спрайт) смогу подтянуть уже только, как запулю себе изменения из мейн.

kateryna-klimchuk committed:
header-html + html-mob-menu

kateryna-klimchuk committed:
add mob-menu script

kateryna-klimchuk committed:
re-named class

Изменения в лого, переименование классов, стили на хедер, кроме кнопок.

Hanna-Serdjuk commented on src/partials/header.html:
```diff
@@ -1 +1,73 @@
+<header class="site-header header">
+    <nav class="header-nav conteiner">
```
ошибка в написании класса contAiner. У нас есть такой общий контенер для секций, но для хедера он другой, тебе наверное нужно сделать под себя с классом уникальным

kateryna-klimchuk replied:
Создала отдельный контейнер для хедер

Hanna-Serdjuk commented on src/sass/layout/_header.scss:
```diff
@@ -0,0 +1,168 @@
+    justify-content: space-around;
+    align-items: center;
+    margin-left: 30px;
+
```
для этой кнопки на десктопе не нужен левый маржин 30, я так понимаю ты его использовала чтобы отодвинуть кнопку открытия меню. Поставь его авто для десктопа

kateryna-klimchuk replied:
Убрала марджин, спасибо!

Hanna-Serdjuk reviewed (changes requested):
Добавь, пожалуйста, флекс и размести элементы хедера по ширине. Также надо задать высоту хедеру, думаю можно паддингами

kateryna-klimchuk committed:
reviewers changes

Задала свой контейнер хедеру, убрали ненужный марджин, подправила размер мобильного меню
